nix/lib/default.nix

```diff
@@ -348,36 +396,32 @@ in rec {
           done
 
         '';
+
         checkPhase = ''
           export LANG=en_US.UTF-8
           export LOCALE_ARCHIVE=${glibcLocales}/lib/locale/locale-archive
-          for i in src-*; do
-            pushd $i
-            flake8 --exclude=nix_run_setup.py --exclude=build/*
-            # TODO: py.test
-            popd >> /dev/null
-          done
+
+          flake8 --exclude=nix_run_setup.py,migrations/,build/
+          # TODO: py.test
```

Comment:
You can add it already, there are dummy unit tests for releng_clobberer and shipit_dashboard